contracts/test/token/erc20/governance-erc20.ts

```diff
@@ -0,0 +1,785 @@
+import {
+  DAO,
+  DAO__factory,
+  GovernanceERC20,
+  GovernanceERC20__factory,
+  IERC165Upgradeable__factory,
+  IERC20MintableUpgradeable__factory,
+  IERC20PermitUpgradeable__factory,
+  IERC20Upgradeable__factory,
+  IVotesUpgradeable__factory,
+} from '../../../typechain';
+import {getInterfaceId} from '../../../utils/interfaces';
+import {deployWithProxy} from '../../../utils/proxy';
+import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
+import {expect} from 'chai';
+import {ethers} from 'hardhat';
+
+export type MintSettings = {
+  receivers: string[];
+  amounts: number[];
+};
+
+const MINT_PERMISSION_ID = ethers.utils.id('MINT_PERMISSION');
+const governanceERC20Name = 'GovernanceToken';
+const governanceERC20Symbol = 'GOV';
+
+const addressZero = ethers.constants.AddressZero;
+
+describe.only('GovernanceERC20', function () {
```

Comment:
remove `.only`

Reply:
Damn, fixed in b7e0261f484fc3cdc2ea4085dd44a323d572d11c